zerver/actions/message_delete.py

```diff
@@ -86,6 +86,8 @@ def do_delete_messages(realm: Realm, messages: Iterable[Message]) -> None:
         check_update_first_message_id(realm, stream, message_ids, users_to_notify)
 
     event["message_type"] = message_type
+    if user_profile:
```

Comment:
Should we check if users_to_notify already has this id?

Reply:
Changes made